redaxo/src/addons/media_manager/lib/managed_media.php

```diff
@@ -341,13 +341,10 @@ public function setSourcePath($path)
         $this->sourcePath = $path;
 
         $this->asImage = false;
-        if (!isset($this->image['src'])) {
-            return;
-        }
```

Comment:
Warum ist der isset-Check entfallen? Kann es nun nicht zu Notices kommen?

Reply:
ping

Reply:
Ja, da gibt ne warning. Hab wohl übersehen, es in die Abfrage zu setzen. Code ist korrigiert